code/game/objects/structures/maintenance.dm

```diff
@@ -105,3 +105,129 @@ at the cost of risking a vicious bite.**/
 		return
 	hidden_item = I
 	to_chat(user, span_notice("You hide [I] inside the basin."))
+
+#define ALTAR_INACTIVE 0
+#define ALTAR_STAGEONE 1
+#define ALTAR_STAGETWO 2
+#define ALTAR_STAGETHREE 3
+#define ALTAR_TIME 9.5 SECONDS
+
+/obj/structure/destructible/cult/pants_altar
+	name = "strange structure"
+	desc = "What is this? Who put it on this station? And why does it eminate <span class='hypnophrase'>pants energy?</span>"
```

Comment:
```suggestion
	desc = "What is this? Who put it on this station? And why does it emanate <span class='hypnophrase'>pants energy?</span>"
```

Reply:
also, I think mentioning "pants energy" kind of spoils the surprise, but maybe that's just me

Reply:
maybe only cultists should be able to see that it radiates pants energy, since they can see the exact functions of other cult structures (IIRC)?